src/features/metadata-based-view/MetadataQueryAPIHelper.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * @flow strict
+ * @file Versions Sidebar API Helper
+ * @author Box
+ */
+import getProp from 'lodash/get';
+import find from 'lodash/find';
+import includes from 'lodash/includes';
+import API from '../../api';
+
+import { METADATA_FIELD_TYPE_ENUM, METADATA_FIELD_TYPE_MULTISELECT, ITEM_TYPE_FILE } from '../../common/constants';
+
+import type {
+    MetadataQuery as MetadataQueryType,
+    MetadataQueryResponse,
+    MetadataQueryResponseData,
+    MetadataQueryResponseEntry,
+    MetadataQueryResponseEntryMetadata,
+    MetadataTemplateSchemaResponse,
+} from '../../common/types/metadataQueries';
+
+type SuccessCallback = (metadataQueryCollection: Collection) => void;
+type ErrorCallback = (e: ElementsXhrError) => void;
+
+const SELECT_TYPES: Array<'enum' | 'multiSelect'> = [METADATA_FIELD_TYPE_ENUM, METADATA_FIELD_TYPE_MULTISELECT];
```

Comment:
should these be `typeof METADATA_FIELD_TYPE_ENUM | typeof METADATA_FIELD_TYPE_MULTISELECT`?